src/components/Modal.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { slideDown } from './TodoItem';
+
+export const ModalBackground = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1.875rem;
+  background: white;
+  padding: 3.125rem;
+  border-radius: 0.625rem;
+  text-align: center;
+  box-shadow: 0 0.25rem 0.5rem rgba(0, 0, 0, 0.2);
+
+  ${({ $isOpen }) =>
+    $isOpen &&
+    css`
+      animation: ${slideDown} 0.3s ease-out forwards;
+    `}
+`;
+
+const Modal = ({ ModalText, isOpen, onClose }) => {
+  if (!isOpen) return null;
+
+  return (
+    <ModalBackground onClick={onClose}>
+      <ModalContainer $isOpen={isOpen} onClick={(e) => e.stopPropagation()}>
+        <p>{ModalText}</p>
+      </ModalContainer>
+    </ModalBackground>
+  );
+};
```

Comment:
알림까지 모달로 표시해주시다니... 디테일 챙기느라 고생햇서요🤩

지금 `ConfirmModal` 컴포넌트에도 `Modal` 스타일 임포트해서 사용하고 계신데, 
두 모달창이 같은 컨테이너, 같은 스타일을 사용하되 취소, 삭제 버튼에만 차이가 있는 거라면
`조건부 랜더링`을 사용해서 하나의 컴포넌트 내에서 관리해주시면 더 의미있는 컴포넌트가 될 것 같아요!
```suggestion
const Modal = ({ isOpen, onClose, content, isConfirmModal, onConfirm }) => {
  if (!isOpen) return null;

  return (
    <ModalBackground onClick={onClose}>
      <ModalContainer $isOpen={isOpen} onClick={(e) => e.stopPropagation()}>
        <p>{content}</p>
        {isConfirmModal && (
          <ButtonGroup>
            <ModalButton $cancel onClick={onClose}>취소</ModalButton>
            <ModalButton onClick={onConfirm}>삭제</ModalButton>
          </ButtonGroup>
        )}
      </ModalContainer>
    </ModalBackground>
  );
};
```
이런식으로 버튼 텍스트를 `onClose`나 `onConfirm`과 같은 상태에 맞게 동적으로 바꿔서 사용한다면 모달창 관리가 더 쉬워질 것 같아요! 
(물론 `app.js`에서도 추가적인 수정이 필요하겠지만요 ...🥹)